internal/wrapnix/wrapper.go

```diff
@@ -89,22 +94,33 @@ func CreateWrappers(ctx context.Context, devbox devboxer) error {
 //     So, the bin-wrappers need to use a symlink to the latest devbox binary. This
 //     symlink is updated when devbox is updated.
 func CreateDevboxSymlink() (string, error) {
```

Comment:
change definition of this function to return sym link dir.

Reply:
I'd prefer not doing that, since I'd expect a function named `CreateDevboxSymlink` to return the path to the symlink, rather than its directory. Changing the name to `CreateDevboxSymlinkAndGetDir` feels verbose. I think its okay for the handling logic to do `filepath.Dir`

Reply:
but yeah, let me know if you feel strongly, and I can make the change.